server.js

```diff
@@ -4,7 +4,7 @@ var config = require('./config'),
 
 if (arg === 'debug') {
     logger.info('setting loglevel to %s.', arg);
-    logger.transports.console.level = arg;
```

Comment:
Weirdly, yes. 
Using the default logger, in `lib/logger`, seems to create the `logger.transports` object differently, and `console` is undefined, whereas `Console` is there. 
If you create the logger with arguments (as was done up to now) you get `console`.